test-utils/src/commonTest/kotlin/mocks/keyboard/MockFocusHandlerTest.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package mocks.keyboard
+
+import com.splendo.kaluga.base.runBlocking
+import com.splendo.kaluga.test.mock.focus.MockFocusHandler
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class MockFocusHandlerTest {
+
+    @Test
+    fun test_initial_state() = testFocusHandler {
+        assertFalse { it.isFocused }
```

Comment:
Why do you use block rather than just variable check? `assertFalse(it.isFocused)`?

Reply:
[636ffcd](https://github.com/splendo/kaluga/pull/467/commits/636ffcdd7aa90624e114326f7605c008b10ce974)